src/components/RenderRouter/HeroItem.tsx

```diff
@@ -277,19 +278,38 @@ class HeroItem extends React.Component<Props, State> {
             <div className="heroText2">{this.state.content.text6}</div>
             <div className="heroText2">{this.state.content.text7}</div>
             <div className="contactPastorLink">
-              {this.renderLinkButton(this.state.content.button1)}
-
-              {this.state.content.link1Text ? (
-                <div className="heroAContainer">
-                  <Link
-                    className="heroBlackBoxA inverted"
-                    to={this.state.content.link1Action}
-                  >
-                    {this.state.content.link1Text}
-                  </Link>
-                </div>
-              ) : null}
-              {this.state.content.addToCalendar ? (
+              {moment().weekday() === 0 && this.state.locationData.length ? ( // Is Sunday
+                <button
```

Comment:
For the purpose of accessibility (and semantic HTML) this shouldn't be a button.

Re-styled `<LinkButton>` perhaps? Or just a React Router `<Link>`.